content/blog/dplyr-1-0-0/index.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: dplyr 1.0.0 available now!
+output: hugodown::hugo_document
+
+description: > 
+  dplyr 1.0.0 is now available from CRAN!
+author: Hadley Wickham
+date: '2020-06-01'
+slug: dplyr-1-0-0
+
+photo:
+  url: https://unsplash.com/photos/W8BNwvOvW4M
+  author: Helinton Fantin
+
+categories:
+  - package
+tags:
+  - dplyr
+---
+
+I'm very excited to announce the ninth and final blog post in the [dplyr 1.0.0 series](/tags/dplyr): [dplyr 1.0.0](http://dplyr.tidyverse.org/) is now available from CRAN! Install it by running:
```

Comment:
The tag link will go to all dplyr-tagged posts (we'd discussed the possibility of doing dplyr-1-0-0). Either way is fine by me, just making sure this was intentional.